CMakeLists.txt

```diff
@@ -3,6 +3,17 @@ project(mlpack C CXX)
 
 include(CMake/cotire.cmake)
 
+# Validate md5 hash give md5file url and the file directory.
+# If hash check fails set HASH_CHECK_FAIL to non-zero and remove the files.
+macro (check_hash MD5_URL DIR HASH_CHECK_FAIL)
```

Comment:
So for this to work you need the md5 url right?

Reply:
Right, later on `http://www.mlpack.org/files/stb/hash.md5` is used as the `MD5_URL`, so I got that set up on mlpack.org now. :+1: